modules/vault-cluster/variables.tf

```diff
@@ -253,3 +253,9 @@ variable "protect_from_scale_in" {
   type        = bool
   default     = false
 }
+
+variable "capacity_rebalance" {
+  description = "(Optional) Whether capacity rebalance is enabled. Otherwise, capacity rebalance is disabled."
```

Comment:
Can we add something about the impact?  Like "if capacity balance is enabled your EC2 could get terminated/scaled to rebalance AZs".  Or whatever it does.

Reply:
Sure I can add a blurb and link to the AWS docs